crates/networking/src/handlers/gossip.rs

```diff
@@ -34,8 +34,11 @@ impl NetworkService<'_> {
                     .await;
             }
             Subscribed { peer_id, topic } => {
-                let added = with_connected_peers(&topic, |connected_peers| {
-                    connected_peers.fetch_add(1, gadget_std::sync::atomic::Ordering::Relaxed);
+                let added = with_connected_peers(&topic, |_connected_peers| {
```

Comment:
If you're just commenting this out why even assign the variable?

Reply:
In case a bug happens soon as a result of this change (maybe somebody added this code with a specific intention in mind), we can track it down quicker

Reply:
Calling that function also runs some logic, not just the closure